lib/ValidationError.ts

```diff
@@ -55,15 +55,18 @@ function fromZodIssue(
   return issue.message;
 }
 
+export type FromZodErrorZodError = zod.ZodError;
```

Comment:
issue: I am a bit troubled by this one. I am thinking we should rename to `ZodError`. Why? Because (a) this type is controlled by `zod` and (b) it's likely that another function will need it in the future.

@nikoskalogridis what do you think?

Reply:
I agree `ZodError` is better.

Reply:
Okay, thanks. Maybe `FromZodErrorOptions` should be renamed to `ErrorOptions` or `ErrorReportingOptions` as well?

Reply:
Hello, I have renamed `FromZodErrorZodError` to `ZodError`.

Cheers

Reply:
> Okay, thanks. Maybe `FromZodErrorOptions` should be renamed to something like `ErrorOptions`, `ErrorReportOptions` or `ErrorReportingOptions` as well?

Since those options are specific to this function I think its ok